pytroll_collectors/geographic_gatherer.py

```diff
@@ -188,12 +188,10 @@ def run(self):
                         raise RuntimeError
         except KeyboardInterrupt:
             logger.info("Shutting down...")
-        except RuntimeError:
-            logger.critical('Something went wrong!')
-        except OSError:
-            logger.critical('Something went wrong!')
+        except (RuntimeError, OSError) as err:
+            logger.critical('Something went wrong: %s', err)
```

Comment:
Would `logger.exception` be more informative here?  Advantage: more information.  Disadvantage: lower log level (error rather than critical).

Reply:
Hmm. Yes, I think the traceback would be nice, so I'll replace this.